Source/WebCore/dom/TreeScopeInlines.h

```diff
@@ -62,4 +67,17 @@ inline bool TreeScope::containsMultipleElementsWithName(const AtomString& name)
     return m_elementsByName && !name.isEmpty() && m_elementsByName->containsMultiple(name);
 }
 
+// FIXME: Move these to ContainerNodeInlines.h
```

Comment:
Any reason not to do this straight away or is that why this is a draft?

Reply:
I'm trying to limit the scope of this PR. It can grow out of control if I start enforcing that everywhere.